checklist.py

```diff
@@ -450,5 +450,31 @@ def _parse_file(self, fp, root):
         return skbio_imports
 
 
+class FluxCapacitorValidator(RepoValidator):
+    """Ensure that the __future__ statements are fluxing correctly"""
+    reason = ("These files do not have the following import at the start:\n\n"
+              "from __future__ import absolute_import, division,"
+              " print_function\n")
+
+    def _validate(self, root, dirs, files):
+        failures = []
+        expected = set(("absolute_import", "division", "print_function"))
+        for file in files:
+            if file.endswith(".py"):
+                filename = os.path.join(root, file)
+                failures.append(filename)
+                with open(filename) as f:
+                    source = ast.parse(f.read())
+                    for node, _ in zip(ast.iter_child_nodes(source), range(2)):
+                        if isinstance(node, ast.Expr):
+                            continue
+                        if isinstance(node, ast.ImportFrom):
+                            if node.module == "__future__":
+                                if expected <= set(n.name for n in node.names):
```

Comment:
Could also use set comprehension here: `if expected <= {n.name for n in node.names}`

The `<=` is a little confusing here though, what set operation is that? Shouldn't this be set difference, e.g.:

```python
16:21:43 (mcdonadt@Daniels-MacBook-Pro-3):/Users/mcdonadt/ResearchWork/software/scikit-bio
> {1,2,3} - {2,3,4,5,6}
Out[7]: {1}
```

It may be more explicit to use the method as well instead of the overloaded operator